server/lobbyconnection.py

```diff
@@ -339,11 +340,12 @@ def on_connection_made(self, protocol: QDataStreamProtocol, peername: Address):
                             player.lobby_connection.send_warning(message_text)
                         )
                     except AttributeError:
+                        # In case player.lobby_connection was None
                         self._logger.debug("Failed to send broadcast to %s", player)
                     except Exception:
                         self._logger.exception("Failed to send broadcast to %s", player)
 
-                await asyncio.gather(*tasks, return_exceptions=True)
```

Comment:
It's a matter of preference, but I'd catch ConnectionError either directly with `as_completed` or by wrapping each task with a try-catch. Just to avoid that 'pass exception type to function in order to catch it' thing :/

Reply:
I dunno. To me it feels quite similar to the `contextlib.suppress` function. It helps you write your try...catch...ignore block in one line instead of 5 lines. I don't really see a benefit to writing the whole thing out each time when we're just ignoring a specific type of exception.

```python
with contextlib.suppress(AssertionError, ConnectionError):
    assert False
```

I don't think there is inherently anything wrong with passing exception classes as types.